Sources/Slacket/Parsers/ParserType.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  ParserType.swift
+//  Slacket
+//
+//  Created by Jakub Tomanik on 20/05/16.
+//
+//
+
+import Foundation
+import Kitura
+
+// To define models that can be parsed
+protocol Parsable {}
+
+// To define parsers
+protocol ParserType {}
+
+protocol ParserDecoder: ParserType {
+    
+    func parse(body: ParsedBody) -> Parsable
```

Comment:
Are you sure we can do that things? Imo in such case we get an error saying that `SlackCommandDecoder` doesn't conform to protocol `ParseDecoder`. I think we need to introduce associated type to `ParseDecoder` which will conform to `Parsable` protocol